tdp/core/deployment/deployment_plan.py

```diff
@@ -101,22 +137,15 @@ def from_reconfigure(dag, cluster_variables, service_component_deployed_version)
             restart=True,
             filter_expression=r".+_(config|(re|)start)",
             filter_type=FilterTypeEnum.REGEX,
+            state=DeploymentStateEnum.PLANNED,
```

Comment:
I do not see `state` argument for `DeploymentPlan.from_dag` and the state is not needed, the `DeploymentPlan.from_dag` function already put a `PLANNED` state.

Reply:
Done